### ISSUE TITLE
Feature Request-Mod support?

### ISSUE THREAD

**Issue by Zedifus** [SimpleServer/SimpleServer#173]
I don't know if its possible but is there or will there be mod support for SS? without using an alternative jar (craftbukkit), like where mods would be made for simple server and can be added to a folder that SS generates (something like, Mods or plugins)

**Comment by BigBrick** [comment]
Simple server wrapper is a wrapper, nothing more. It isn't actually a server, so no, it can't have mods.

**Comment by acertain** [comment]
no.  well, you CAN mod SS, but it will add functionality like bukkit plugins.  No support.

Modding is just modifying the codes(source).

but you CAN mod the normal MC MP jar that SS uses like any other MC MP jar.

**Comment by BigBrick** [comment]
I misread. I thought you were talking about something different. 
Trust me, I'm well aware of what modding is.
Again, I am also aware that you can mod the regular .jar.

**Comment by acertain** [comment]
lol, i was not the OP, and was just clarifying.

**Comment by BigBrick** [comment]
This entire conversation has been misunderstandings :P
